src/util/path.ts

```diff
@@ -2,26 +2,133 @@ import * as vscode from 'vscode';
 import * as path from 'path';
 import untildify = require('untildify');
```

Comment:
Add tests for these utils?

Reply:
Refactored the path util to be more unit testable, but one method is still hard to test since it depends on the platform.